apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/service/CommitService.java

```diff
@@ -35,9 +35,18 @@ public CommitService(final CommitRepository commitRepository) {
   }
 
   @Transactional
-  public Commit save(Commit commit){
+  public void createCommit(String appId, String clusterName, String namespaceName, String configChangeContent,
```

Comment:
Your point makes sense, but I think having five input parameters here is still acceptable because there is one parameter, "operator," that is set twice.